Doc/library/unittest.mock.rst

```diff
@@ -1369,7 +1369,8 @@ patch
     "as"; very useful if :func:`patch` is creating a mock object for you.
 
     :func:`patch` takes arbitrary keyword arguments. These will be passed to
-    the :class:`Mock` (or *new_callable*) on construction.
+    :class:`AsyncMock` if the patched object is an async function, to 
```

Comment:
Is this true only for async functions? I thought this was true for all "async things"? @lisroach ?

Reply:
Yes it would be more correct to say "if the patched object is asynchronous" to cover all async object types

Reply:
Glad to know, will adjust the wording on this. Just curious ab out what other async things there in Python? generators?!

Reply:
Yep, Async Generators: https://www.python.org/dev/peps/pep-0525/
Iterators, context managers: https://www.python.org/dev/peps/pep-049
Plus there are "coroutine objects" but those are going away, along with some older terminology. Best just to cover all the bases with a broad statement ;)